scripts/update_meilisearch_version.sh

```diff
@@ -201,7 +200,7 @@ echo "${INFO_LABEL}Stopping MeiliSearch Service to update the version."
 ## Stop meilisearch running
 systemctl stop meilisearch # stop le service pour pouvoir changer la version
 
-## Move le nouveau binaire
+## Move the new binary
```

Comment:
Wouldn't it be to move the previous/current binary?

Reply:
```suggestion
## Move the binary of the current version to the temp folder
```